mne/commands/mne_anonymize.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+"""Anonymize .fif file.
+
+You can do for example:
+
+$ mne anonymize sample_audvis_raw.fif
+"""
+
+# Authors : Dominik Krzeminski
+
+import sys
+import mne
+import os
+
+ANONYMIZE_FILE_PREFIX = 'anon'
+
+
+def mne_anonymize(fif_fname, out_fname, dir_name="", overwrite=False):
+    """ Call *anonymize_info* on fif file and save.
+    Parameters
+    ----------
+    fif_fname : str
+        Raw fif File
+    out_fname : str
+        Output file name
+    dir_name : str
+        Path to directory with files
```

Comment:
I would just rely on fif_fname. You can split the fif_fname in basedir and file using os module.